xcarchive/macos.go

```diff
@@ -43,7 +43,7 @@ func newMacosBaseApplication(path string) (macosBaseApplication, error) {
 
 	var provisioningProfile *profileutil.ProvisioningProfileInfoModel
 	{
-		provisioningProfilePath := filepath.Join(path, "Contents/Resources/embedded.mobileprovision")
+		provisioningProfilePath := filepath.Join(path, "Contents/embedded.provisionprofile")
```

Comment:
Is it for all versions?

Reply:
About the extension I'm sure, but about the location... 🤔 
I have to check it with Xcode 9 too, just to be sure 😭